### PR TITLE
Added agdpmod=pikera to Navi 23

### PR DESCRIPTION
**Pull request by perez987**
- Added `agdpmod=pikera` to Navi 23 since many if not all Navi 21 and 23 cards require this boot arg to avoid black screen on Desktop.
<s>- Added a spoiler to inform that in Ventura it is possible to use Navi 21 and 23 cards without WhateverGreen and the boot arg by setting the SMBIOS model to iMacPro1,1 or MacPro7,1.
- Removed XFX from the list of brands to avoid on Polaris cards, many users are using them with no issues from Catalina to Ventura.</s>
- Changed PowerColour to PowerColor (typo)
- Added 14 to Navi 10 boot arg comment (@1Revenger1).

### CODE REVIEW

File: modern-gpus/amd-gpu.md [hunk 1]
```diff
@@ -35,13 +37,20 @@ Supported Cards:
...
 
 * [Lilu.kext](https://github.com/acidanthera/Lilu/releases)
 * [WhateverGreen.kext](https://github.com/acidanthera/WhateverGreen/releases)
 
+<details>
+  <summary>Navi 21 and 23 series without WhateverGreen</summary>
+  
+ On macOS Ventura, by setting iMacPro1,1 or MacPro7,1 SMBIOS you can use Navi 21 and 23 cards without WhateverGreen or `agdpmod=pikera` in boot args.
+  
+</details>
```
Comment (1Revenger1) [inline modern-gpus/amd-gpu.md:52]:
```suggestion
```
This works for other macOS versions and other cards too. I did this with my GTX 1080 in High Sierra.
The main downside is that you'll be missing other patches, especially on GCN cards. It's probably best to not advise this. This really isn't the right place for something like this anyways.

In the future, dropdowns should use
```md
::: details <name>

<body>

:::
```

File: modern-gpus/amd-gpu.md [hunk 2]
```diff
@@ -130,7 +139,7 @@ Needed kexts:
 
 Regarding Polaris, basically every model of card is supported as long as it’s running either a Polaris or Baffin core. Lower end cards like the RX 550 may run a Lexa core, meaning no support in macOS.
 
-The only brands **you should avoid with the Polaris series would be XFX, PowerColour, HIS and VisionTek** as many users have had bootloader and macOS boot issues. Other users have found fixes/workarounds, though nothing consistent. This seems to be caused by having an odd VBIOS that doesn't communicate well with macOS and the only real solution is flashing another VBIOS, which is not ideal for most users.
+The only brands **you should avoid with the Polaris series would be PowerColor, HIS and VisionTek** as many users have had bootloader and macOS boot issues. Other users have found fixes/workarounds, though nothing consistent. This seems to be caused by having an odd VBIOS that doesn't communicate well with macOS and the only real solution is flashing another VBIOS, which is not ideal for most users.
```
Comment (1Revenger1) [inline modern-gpus/amd-gpu.md:142]: Specifically the XFX 460/560 models had issues. Probably best to note this somewhere.

File: modern-gpus/amd-gpu.md [hunk 3]
```diff
@@ -65,7 +74,7 @@ Radeon Pro:
 * Radeon Pro W5500
 * Radeon Pro W5700
 
-Note: Most Navi cards currently require the boot argument `agdpmod=pikera` to get proper display out.
+Note: Most Navi 10 cards currently require the boot argument `agdpmod=pikera` to get proper display out.
```
Comment (1Revenger1) [inline modern-gpus/amd-gpu.md:77]:
```suggestion
Note: Most Navi 10 and 14 cards currently require the boot argument `agdpmod=pikera` to get proper display out.
```
5500/XT is Navi 14

**Review by 1Revenger1 — CHANGES_REQUESTED** [review]

File: modern-gpus/amd-gpu.md (hunk 1 above)
Comment (perez987) [reply modern-gpus/amd-gpu.md:52]: Thanks. Noted the details block, I thought that spoiler github block was ok. I’ll review your comments later.

File: modern-gpus/amd-gpu.md (hunk 3 above)
Comment (perez987) [reply modern-gpus/amd-gpu.md]: Updated. Thanks.

**Review by 1Revenger1 — APPROVED** [review]
LGTM

**Pull request merged**